woocommerce/woocommerce-ios
# Order list: Add entry point to create test order

Part of #10292 


## Description

This PR adds a new entry point on the empty order list for launched stores following design Fm8xqZRLsZywfBLyRrOCMV-fi-1_5174. This entry point navigates to a new screen with instructions to create a test order to get new merchants started.

Changes:
- The empty state of the order list was updated with new wording and button.
- New screen for create test order with instructions and a button to open an external browser.

Notes: 
- The empty state image does not change (p1690255932688589/1689926465.212349-slack-C0354HSNUJH)
- We don't have the link to show detailed instructions yet, so the Learn more link is not added for now.
- Analytics will be handled in a subsequent PR.

## Testing instructions

1. Testing stores that are not launched yet:
Log in to a trial WooExpress site and navigate to the order list, the empty state should be as before:

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/95a7afa0-fde8-41fd-b6ce-4fa7f98313f9" width=320 />

2. Testing launched stores:
- Log in to a store that has been launched but has no order yet.
- Navigate to the Orders tab, notice that the empty state is displayed with wording and CTA for creating test orders.
- Tap the button on the empty state, the app should navigate to a new screen with instructions to create test orders.
- Tap the button on the create test order screen, you should be navigated to an external browser with your test store opened.
 

## Screenshots

| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/51051f43-2d7d-4ec0-8139-12e08abf62b3" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/f77e317d-28c5-45ab-9d2f-55881adf92ac" width=320 /> |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/95c8b80a-f24d-4d5e-a7f5-9f6cdefe2fa2" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/2b7fcae5-e452-4306-a956-54e3f7c1ad2d" width=320 /> |





---
